### ISSUE TITLE
Error thrown on accessing special characters dropdown with keyboard

### ISSUE THREAD

**Issue by niegowski** [ckeditor/ckeditor5#10423]
## 📝 Provide detailed reproduction steps (if any)

1.  http://fake.ckeditor.com:8081/build/docs/ckeditor5/latest/features/special-characters.html
2.  press <kbd>Alt</kbd>+<kbd>F10</kbd>
3.  navigate to special characters dropdown
4.  press <kbd>enter</kbd> or <kbd>down</kbd> arrow to open dropdown
5.  press <kbd>down</kbd> arrow key

### ✔️ Expected result

Should not throw and probably allow to pick special character with keyboard.

### ❌ Actual result

```
snippet.js:4 Uncaught TypeError: this.children.first.focus is not a function
    at s.focus (snippet.js:4)
    at snippet.js:4
    at Object._listener.listenTo.priority (snippet.js:4)
    at Object.fire (snippet.js:4)
    at c.<anonymous> (snippet.js:4)
    at c.fire (snippet.js:4)
    at HTMLDivElement.n (snippet.js:4)
```

![](https://user-images.githubusercontent.com/1232187/130808064-4053f53e-6661-473e-b287-fc45a3b6a8ca.gif)

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

*   Browser: …
*   OS: …
*   First affected CKEditor version: reproducible since v18
*   Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Reinmar** [comment]
Note: The editor still works after this error is thrown.

Plus: This never worked.

Plus: #9037.

**Comment by FilipTokarski** [comment]
This should be checked after the latest enhancements in focus handling.

**Comment by RoopanV** [comment]
The same problem with Tables dropdown too.

**Comment by mlewand** [comment]
I believe this issue should be fixed in the upcoming release, isn't it?

**Comment by Mgsy** [comment]
> I believe this issue should be fixed in the upcoming release, isn't it?

Yes, it's fixed.
